medusa/server/api/v2/config.py

```diff
@@ -187,11 +233,10 @@ class DataGenerator(object):
     @classmethod
     def sections(cls):
         """Get the available section names."""
-        return [
-            name[5:]
-            for (name, function) in inspect.getmembers(cls, predicate=inspect.isfunction)
-            if name.startswith('data_')
-        ]
+        cls.data__ = [name[5:] for (name, function) in inspect.getmembers(cls, predicate=inspect.isfunction) if
+                      name.startswith('data_')]
+        cls.cls_data__ = cls.data__
+        return cls.cls_data__
```

Comment:
@p0psicles 
Why did you make this change?

Reply:
I didnt. I think someone else did.. And ik got in here through a rebase